app/javascript/stylesheets/components/_notes.scss

```diff
@@ -1,3 +1,7 @@
 .notes-table-height {
   height: calc(100vh - 160px);
 }
+
+.notes-font {
+  font-family: 'SF Pro Text';
+}
```

Comment:
I don't think this is needed. Was it mentioned in the Figma design to specifically add this font?

Reply:
Yes it was mentioned in the Figma design to use that particular font. Although I also thought that it was less appealing than the default design. I will remove these as well.

Reply:
Let's not add custom fonts. Similar to: https://github.com/navaneethsdk/wheel/pull/9#discussion_r977295145